tests/disaster-recovery/regional-dr/test_managed_cluster_node_failure.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import time
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier4, tier4b
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.ocs.resources.pod import (
+    Pod,
+    get_pod_node,
+    get_pods_having_label,
+    wait_for_pods_to_be_running,
+    wait_for_pods_to_be_in_statuses,
+)
+from ocs_ci.ocs.utils import get_non_acm_cluster_config
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
+@tier4b
+@pytest.mark.parametrize(
+    argnames=["resource_on_node"],
+    argvalues=[
+        pytest.param(
+            "ramen-dr-cluster-operator", marks=pytest.mark.polarion_id("OCS-4438")
+        ),
+        pytest.param("rbd-mirror", marks=pytest.mark.polarion_id("OCS-4437")),
+        pytest.param("submariner-gateway", marks=pytest.mark.polarion_id("OCS-4439")),
```

Comment:
I think it's good to add scenarios for other submariner pods too, as we have been seeing a lot of connectivity issues due to submariner pods which may not be specific to gateway only.